dolphin-dnn/src/main/java/edu/snu/dolphin/dnn/conf/LayerConfigurationParameters.java

```diff
@@ -64,4 +64,27 @@
   @NamedParameter(doc = "loss function of loss layer", short_name = "lossFunc")
   public static final class LossFunction implements Name<String> {
   }
+
+  /**
+   * For pooling layers.
+   */
+  @NamedParameter(doc = "pooling type of pooling layer")
+  public static final class PoolingType implements Name<String> {
+  }
+
+  @NamedParameter(doc = "stride height of pooling / convolutional layer", short_name = "strideH")
```

Comment:
@beomyeol Come to think of it, do we need short names for these parameters? Everything's being done by protobuf right now.